src/Types/MutationType.php

```diff
@@ -36,20 +34,20 @@ public function __construct(TypeLoader $typeLoader)
         foreach (ClassFinder::models() as $name => $classname) {
             $model = new $classname();
             $fields['create' . $name] = $this->create($name, $model, $typeLoader);
-            $fields['update' . $name] = $this->update($name, $typeLoader);
-            $fields['updateMany' . $name . 's'] = $this->updateMany($name, $typeLoader);
-            $fields['delete' . $name] = $this->delete($name, $typeLoader);
-            $fields['restore' . $name] = $this->restore($name, $typeLoader);
-            $fields['import' . $name . 's'] = $this->import($name, $typeLoader);
-            $fields['import' . $name . 'sAsync'] = $this->importAsync($name, $typeLoader);
-            $fields['export' . $name . 'sAsync'] = $this->exportAsync($name, $model, $typeLoader);
+            $fields['update' . $name] = $this->update($name);
```

Comment:
wäre ja schon fast möglich wie in laravel mit den ganzen magic ala
```php
$queryFunctions = ['create', 'update', ...];
foreach (ClassFinder::models() as $name => $classname) {
  foreach (queryFunctions as $fnName) {
    $fields[$fnName . $name] = $this->$fnName($name);
    ...
```
nur plural + async machen da probleme :laughing: 

Hier aber bringts wirklich nix, machts eher komplizierter

Reply:
Es wird hier sicher in Zukunft noch Änderungen geben - aber ein PR soll ja möglichst unter 200 Zeilen haben, und ich hab hier schon jetzt schon 450 Zeilen - darum wird's in diesem PR jetzt, hier keine zusätzlichen Änderungen mehr geben.